bin/install.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# This script downloads the latest release version of EAS CLI from GitHub and
+# installs it in /usr/local/bin
+
+{
+  # All code is inside a block to ensure the script executes only when
+  # downloaded completely.
+
+  set -e
```

Comment:
```suggestion
  set -euo pipefail
```

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
(leave off `x`)